### PR TITLE
View changes

### PR DESCRIPTION
**Pull request by jforgue**

### CODE REVIEW

File: app/views/bus_stops/by_sequence.haml [hunk 1]
```diff
@@ -1,13 +1,16 @@
 .page-title Route #{@route.number} Bus Stops
 
-.container
+%div
+  = link_to 'View by status', by_status_bus_stops_path(number: @route.number)
+- if @collection.first.first.nil?
```
Comment (dfaulken) [inline app/views/bus_stops/by_sequence.haml]: Could you explain `.first.first` to me?

File: app/views/bus_stops/by_status.haml [hunk 2]
```diff
@@ -1,5 +1,7 @@
 .page-title Route #{@route.number} Bus Stops
 
+%div
+  = link_to 'View by sequence', by_sequence_bus_stops_path(number: @route.number)
```
Comment (dfaulken) [inline app/views/bus_stops/by_status.haml]: Let's come up with a better name. 'View by route order', maybe?

File: app/views/bus_stops/by_sequence.haml [hunk 3]
```diff
@@ -1,13 +1,16 @@
 .page-title Route #{@route.number} Bus Stops
 
-.container
+%div
```
Comment (dfaulken) [inline app/views/bus_stops/by_sequence.haml]: Why do we want this link in a div?

Comment (jforgue) [reply app/views/bus_stops/by_sequence.haml]: It was the simplest way to keep it out of line with the next text, other than a few `br` tags .

File: app/views/bus_stops/by_sequence.haml (hunk 1 above)
Comment (mboneil10) [reply app/views/bus_stops/by_sequence.haml]: He wanted to check that `stops` wasn't empty. Probably better to put `@stops = @route.bus_stops` in the controller and rewrite the if?

Comment (mboneil10) [reply app/views/bus_stops/by_sequence.haml]: Hm. I talked to @jforgue and it seems like I'm wrong. I'll let him explain.

**Pull request merged**